integration/spark2/src/main/scala/org/apache/spark/sql/hive/CarbonRelation.scala

```diff
@@ -22,6 +22,7 @@ import scala.Array.canBuildFrom
 import scala.collection.JavaConverters._
 import scala.util.parsing.combinator.RegexParsers
 
+import org.apache.spark.sql.CarbonEnv
```

Comment:
?
Only add an import without changing the code?

Reply:
I will remove it